src/services/codefixes/importFixes.ts

```diff
@@ -564,17 +564,21 @@ namespace ts.codefix {
             : undefined;
     }
 
-    export function getImportKind(importingFile: SourceFile, exportKind: ExportKind, compilerOptions: CompilerOptions): ImportKind {
+    /**
+     * @param forceImportKeyword Indicates that the user has already typed `import`, so the result must start with `import`.
+     * (In other words, do not allow `const x = require("...")` for JS files.)

```

Comment:
```suggestion
     * @param forceImportKeyword Indicates that the user has already typed `import`, so the result must start with `import`.
     *                           For example, we should not provide `const x = require("...")` for JS files if it is clear that a user is writing an `import`.
```

Reply:
I would apply this except it has a bunch of leading spaces?